### PR TITLE
PLNSRVCE-1598: update sprayproxy repository with konflux-ci org

### PR DESCRIPTION
**Pull request by jkhelil**

### CODE REVIEW

**Comment by jkhelil** [comment]
@Roming22

**Review by Roming22 — APPROVED** [review]

**Comment by jkhelil** [comment]
@enarha do yo have an idea why the yaml check is failing ?

**Comment by hugares** [comment]
> @enarha do yo have an idea why the yaml check is failing ?

The container "sprayproxy" is using an invalid container image, "ko://github.com/redhat-appstudio/sprayproxy". Please use images that are not blocked by the BlockList criteria : [".*:(latest)$" "^[^:]*$" "(.*/[^:]+)$"]
object: sprayproxy/sprayproxy apps/v1, Kind=Deployment

**Comment by enarha** [comment]
@jkhelil Please split that PR into two. One to update staging, wait for it to be merged and deployed and verify the change before opening another PR to update production (this is a practice we follow within the team).

**Comment by hugares** [comment]
> @hugares We are using standard kustomization construction https://github.com/redhat-appstudio/infra-deployments/blob/main/components/sprayproxy/staging/kustomization.yaml#L8-L11 where the `name` is actually replaced by the `newName:newTag`. `ko://github.com/konflux-ci/sprayproxy` is just a placeholder replaced by the actual name and tag. We can workaround that by changing that name both in infra-deployments and the sprayproxy repo, by appending some dummy tag, but that will make it more confusing for other people who will look at it. A comment could help, but the preferred workaround for me will be a comment on the problematic line to make the linter skip that error.

I know what is the issue, the kustomize replacement is no longer working... let me put a comment in the diff about it

File: components/sprayproxy/production/kustomization.yaml [hunk 1]
```diff
@@ -2,11 +2,11 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
+  - https://github.com/konflux-ci/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
   - pipelines-as-code-secret.yaml
 
 images:
-  - name: ko://github.com/redhat-appstudio/sprayproxy
+  - name: ko://github.com/konflux-ci/sprayproxy
```
Comment (hugares) [inline components/sprayproxy/production/kustomization.yaml:9]: The manifest in https://github.com/konflux-ci/sprayproxy/config at commit c6f8bf26007c140d51326760e16eb52ae5072e3b are still using `ko://github.com/redhat-appstudio/sprayproxy` as image name so kustomize is no longer replacing the name and tag.

This line must be reverted until manifest in sprayproxy use new org name
```suggestion
  - name: ko://github.com/redhat-appstudio/sprayproxy
```

Comment (enarha) [reply components/sprayproxy/production/kustomization.yaml:9]: Good catch. Thank you.

**Comment by jkhelil** [comment]
I can see this error on e2e
oc apply -f deploy/member-operator/appstudio-dev/ -n toolchain-member-operator || true
error: the path "deploy/member-operator/appstudio-dev/" does not exist
oc apply -f deploy/crds/

I think it is not related to the change
I am retrying the test again
/test appstudio-e2e-tests

**Comment by jkhelil** [comment]
@enarha @hugares @Roming22 
I think we can approve this now ?

**Comment by jkhelil** [comment]
@zregvart @mmorhun Can you aprove this please ?

**Pull request merged**